Library/Formula/srmio.rb

```diff
@@ -15,7 +15,7 @@ class Srmio < Formula
 
   def install
     if build.head?
-      system "chmod u+x genautomake.sh"
+      chmod 1, "genautomake.sh"
```

Comment:
Excellent. @UniqMartin, thanks for the explanation. Read and execute for everyone, write only for the owner - makes sense. I overlooked this partly because I wasn't sure it was OK to diverge much from the original code, but `chmod 0755` likely aligns with the original intention, anyway - as you pointed out.

I've updated my PR. :balloon: